hoot-josm/pom.xml

```diff
@@ -0,0 +1,101 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 
+	http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>hoot.core.josm</groupId>
+	<artifactId>hoot-josm</artifactId>
+	<packaging>jar</packaging>
+	<version>0.0.1</version>
+	<name>hoot-josm</name>
+	<url>http://hootenanny.org</url>
+
+	<properties>
+		<jdk.version>1.8</jdk.version>
+                <josm.artifactId>josm</josm.artifactId>
+		<!-- josm.artifactId>me-josm</josm.artifactId -->
+		<josm.version>15628</josm.version>
+		<!-- josm.version>4.4.9</josm.version -->
+	</properties>
+
+  <repositories>
+    <repository>
+      <id>org.openstreetmap.josm</id>
+      <url>https://josm.openstreetmap.de/nexus/content/repositories/releases/</url>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+    </repository>
+  </repositories>
+
+	<dependencies>
+                <dependency>
+    			<groupId>org.openstreetmap.josm</groupId>
```

Comment:
Which external JOSM gets pulled down can be controlled here. The `hoot-josm` qmake post step calling Maven relies on this. devops will need to do some file string replacement here to package our internal josm.